packages/simple-markdown/src/index.js

```diff
@@ -25,12 +26,11 @@
 /// <reference lib="ES2018" />
 /// <reference path="../simple-markdown.d.ts" />
```

Comment:
Given that we're using regular Flow type annotations in this file now I don't think TypeScript checking will work any more since it won't be able to parse some of the syntax.  We should probably remove these line, the `@ts-check` comment at the top, as well as the simple-markdown.d.ts file itself.

Reply:
Good suggestion. Let me do Flow conversion to real types and removal of typescript as one bite